plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcPageSourceFactory.java

```diff
@@ -252,13 +263,13 @@ private ConnectorPageSource createOrcPageSource(
 
         boolean originalFilesPresent = acidInfo.isPresent() && !acidInfo.get().getOriginalFiles().isEmpty();
         try {
-            FileSystem fileSystem = hdfsEnvironment.getFileSystem(identity, path, configuration);
```

Comment:
`HdfsEnvironment hdfsEnvironment` is unused now

Reply:
It is used on line 414